express-crud/server.ts

```diff
@@ -0,0 +1,135 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const { grades } = req.body;
```

Comment:
I don't understand this. For a GET request, there won't be a body. So there's no need to get `grades`. You shouldn't need to use `grades` in your SQL.